db/migrate/20220720075226_remove_user_id_index_on_cohort_memberships.rb

```diff
@@ -0,0 +1,6 @@
+class RemoveUserIdIndexOnCohortMemberships < ActiveRecord::Migration[7.0]
+  def change
+    remove_foreign_key :cohort_memberships, :users, column: :user_id
+    remove_index :cohort_memberships, :user_id, if_not_exists: true
+  end
+end
```

Comment:
I had some issues with getting the migrations working when I combined things. It basically prevented me from doing rollbacks, which was a bit unfortunate. Once we're happy with the PR, I could try to merge the migrations into one if desired.